boba/cli.py

```diff
@@ -16,15 +17,23 @@
               default='./template.py', show_default=True)
 @click.option('--out', help='Output directory',
               default='.', show_default=True)
-@click.option('--lang', help='Language, can be python/R [default: inferred from file extension]',
+@click.option('--lang', help='Language [default: inferred from file extension]',
               default='')
-def compile(script, out, lang):
+@click.option('--supported_langs', '-sl', 'supp_langs', help='json file with supported languages',
+              default=None)
+def compile(script, out, lang, supp_langs):
```

Comment:
Supplying `--supported_langs` every time when the user compiles and runs a script is tedious.

Is it possible to write the path of the json file in the `BOBA_CONFIG` block? Something similar to the visualizer config here: https://github.com/uwdata/boba/blob/8e009dc6298b094c43f3670f5524350399b92cde/example/hurricane/repro.R#L41